app/bundles/PluginBundle/Integration/AbstractIntegration.php

```diff
@@ -80,7 +80,7 @@ abstract class AbstractIntegration implements UnifiedIntegrationInterface
     /**
      * Used for notifications.
      */
-    protected ?array $adminUsers;
+    protected ?\Doctrine\ORM\Tools\Pagination\Paginator $adminUsers = null;
```

Comment:
PHPSTAN is requesting to set types in the Paginator. Something like `@var Paginator<User>`

Reply:
Done